images/runtime/php-fpm/7.2/base.stretch.Dockerfile

```diff
@@ -98,14 +98,6 @@ RUN { \
                 echo 'zend_extension=opcache'; \
     } > /usr/local/etc/php/conf.d/php.ini
 
-# Install the Microsoft SQL Server PDO driver on supported versions only.
-#  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
-#  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
-RUN set -eux; \
```

Comment:
just curious, why this is not needed anymore?

Reply:
When I try to run it, it shows that this feature was deprecated for 7.2.. so we're going to remove it from the template.